cr3qt/data/docx.css

```diff
@@ -0,0 +1,103 @@
+body { text-align: left; margin: 0; text-indent: 0px }
+
+p { $def.all } 
+
+empty-line { height: 1em }
+
+a { display: inline; $link.all }
+a[type="note"] { $footnote-link.all }
+
+image { text-align: center; text-indent: 0px; display: block }
+p image { display: inline }
+li image { display: inline }
+
+li { display: list-item; text-indent: 0em;  }
+ol { display: block; list-style-type: decimal; margin-left: 1em }
+
+title.h1 p, title.h2 h2 p { 
```

Comment:
This additional `h2` between `title.h2` and `p` looks like a typo.

Reply:
Yes, it was typo